assets/js/theme/global/quick-search.js

```diff
@@ -33,6 +33,7 @@ export default function () {
     };
 
     // stagger searching for 200ms after last input
+    const debounceWaitTime = 1200;
```

Comment:
Can you please also update the value in the comment?

Reply:
thanks @yurytut1993 updated the value in changelog.